lightblue-ldap-common/src/main/java/com/redhat/lightblue/common/ldap/LdapErrorCode.java

```diff
@@ -37,6 +37,9 @@
     /** The LDAP request failed. */
     public static final String ERR_LDAP_REQUEST_FAILED = "ldap:RequestFailed";
 
+    /** LDAP returned an unsuccessful response code. */
+    public static final String ERR_LDAP_UNSUCCESSFUL_RESPONSE = "ldap:UnsuccessfulResponse";
```

Comment:
What's the difference between a `REQUEST_FAILED` and an `UNSUCCESSFUL_RESPONSE`?